api-reference/10 UI Components/dxNumberBox/1 Configuration/format.md

```diff
@@ -8,13 +8,114 @@ default: ''
-
 #include common-demobutton with {
     url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/NumberBox/Formatting/"
 }
 
+The format property can accept the following value types:
+
+- Predefined format    
+
+        <!-- tab: JavaScript -->
+        const number = 1.2345;
+
+        type: "fixedPoint" // 1
+        type: "decimal" // 1.2345
+        type: "percent" // 123%
+        type: "currency" // $1
+
+    You can specify the [precision](/Documentation/ApiReference/Common/Object_Structures/format/#precision) property for decimal numbers.
+
+    If you want to apply a different format, you can implement the [formatter](/api-reference/50%20Common/Object%20Structures/format/formatter.md '/Documentation/ApiReference/Common/Object_Structures/format/#formatter') and [parser](/api-reference/50%20Common/Object%20Structures/format/parser.md '/Documentation/ApiReference/Common/Object_Structures/format/#parser') functions or specify a custom format string as described below. For example, you can use this functionality to enable [large number formats](/Documentation/ApiReference/Common/Object_Structures/format/#type).
+
+- Custom format string    
+
+    Use Unicode Locale Data Markup Language (LDML) patterns to specify a custom format string. An LDML pattern consists of wildcard characters and characters displayed as is. The **format** property supports the following wildcard characters:
+
+    <table class="dx-table">
+        <tr>
+        <th>Format character</th>
+        <th>Description</th>
+        </tr>
+        <tr>
+        <td>0</td>
+        <td>A digit. Displays '0' if the formatted number doesn't have a digit in that position.</td>
+        </tr>
+        <tr>
+        <td>&#35;</td>
+        <td>
+                Any number of leading digits, a single digit, or nothing. If this character goes first in the format string, it can match multiple leading digits (before the decimal point). Consequent characters match a single digit. If the formatted number doesn't have a digit in the corresponding position, displays nothing.</br> 
+                For example, if you apply format "#0.#" to "123.45", the result is "123.4".
+        </td>
+        </tr>
+        <tr>
+        <td>.</td>
+        <td>
+                A decimal separator. </br>
+                Actual character depends on locale.
+        </td>
+        </tr>
+        <tr>
+        <td>,</td>
+        <td>
+                A group separator. </br>
+                Actual character depends on locale.
+        </td>
+        </tr>
+        <tr>
+        <td>%</td>
+        <td>
+                The percent sign. Multiplies the input value by 100.
+        </td>
+        </tr>
+        <tr>
+        <td>;</td>
+        <td>Separates positive and negative format patterns. This character is optional. If you do not use it, then you specify a common format for positive and negative numbers. Negative numbers simply display a minus ("-") prefix. </td>
```

Comment:
`Negative numbers simply display a minus ("-") prefix.`

Is this sentence an example of a common format for negative numbers? In other words, do we need to add "For example" to the beginning of this sentence? Please clarify.

Reply:
I'm going to rewrite this section and add an example, so it becomes more clear